Frontend/app/user/list/list.component.ts

```diff
@@ -7,5 +10,43 @@ import { Component, Input } from '@angular/core';
 })
 
 export class UserListComponent {
-  @Input() users;
+  @Input('users') users;
+
+  private selected:string;
+  private classCode:string;
+
+  constructor(private classService: ClassService, private toastr: ToastrService, private location: Location){}
+
+  ngOnInit(){
+    let path = this.location.path().split('/')
+    this.classCode = path[path.length-1]
```

Comment:
I would have to look at the code outside of this PR to have a better opinion but instead of parsing for the classCode could we just bind it to another `@Input` attribute?